webbpsf/opds.py

```diff
@@ -186,22 +185,22 @@ def __sub__(self, x):
         return output
 
     def as_fits(self, include_pupil=True):
-        """ Return an OPD as a fits.HDUList object
+        """ Return the OPD as a fits.HDUList object
 
         Parameters
         -----------
         include_pupil : bool
             Include the pupil mask as a FITS extension?
         """
 
-        output = fits.HDUList([self._opdHDU])
-        output[0].header.update('EXTNAME', 'OPD')
+        output = fits.HDUList([fits.ImageHDU(self.opd)])
+        output[0].header['EXTNAME'] = 'OPD'
+        output[0].header['BUNIT'] = 'meter'  # Will this always be the case?
```

Comment:
> Will this always be the case? 

I think so. `poppy` internally works with all OPDs in SI units, i.e. meters. It is capable of reading in FITS files with OPDS in other units, like microns or nanometers, but it immediately converts the arrays into meters. (That's all inside `poppy.poppy_core.FITSOpticalElement.__init__`). For output, I don't see any need right now for a feature that would let us output in other units, so I think this is good as-is.